### PR TITLE
Asciidoc => AsciiDoc

### PR DESCRIPTION
**Pull request by mojavelinux**
- fix capitalization of AsciiDoc
- AsciiDoc is UTF-8 compliant, not ASCII

### CODE REVIEW

**Comment by mojavelinux** [comment]
Sure thing. Updated!

File: README.adoc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 = Writing Gradle Tasks image:https://travis-ci.org/{repo-path}.svg?branch=master["Build Status", link="https://travis-ci.org/{repo-path}"] 
+:repo-path: gradle-guides/writing-getting-started-guides
```
Comment (ysb33r) [inline README.adoc]: Thanks for spotting this. It was a left-over from an older migration.

**Comment by ysb33r** [comment]
Thank you for the contribution!

**Comment by mojavelinux** [comment]
:+1:

**Pull request merged**